opcode.def

```diff
@@ -10,3 +10,4 @@ jge,        7,       1,       1,          0
 jgt,        8,       1,       1,          0
 jnz,        9,       1,       1,          0
 jmp,       10,       1,       0,          0
+mul,       11,       1,       1,          1
```

Comment:
Move `mul` to the line right after `add` and `sub`. Arithmetic type instructions should be in the same group.